imports/plugins/included/actionableAnalytics/client/components/InventoryReport.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { registerComponent } from "@reactioncommerce/reaction-components";
+import { SearchInput } from "../components/input/SearchInput";
+import { InventoryTable } from "../components/Table/InventoryTable";
+
+export const InventoryReport = (props) => {
+  const { productInventoryData, selectedTab } = props;
```

Comment:
You can destructure the props you need in the params so that you won't need this `const`. I.e.
```js
export const InventoryReport = ({ productInventoryData, selectedTab, handleInventoryInputChange }) => (
  <div>
    ...
  </div>
);